2022-RapidReact/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -34,7 +39,14 @@ public RobotContainer() {
    * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then passing it to a {@link
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
-  private void configureButtonBindings() {}
+  private void configureButtonBindings() {
+
+  final JoystickButton yButton = new JoystickButton(r_xbox, 4);
+  final JoystickButton aButton = new JoystickButton(r_xbox, 1);
+
+  yButton.whileHeld(g_balls);
+  aButton.whileHeld(i_motor);
+  }
```

Comment:
pensando agora, fica mais simples para o piloto não ter que ficar segurando vários botões ao mesmo tempo. Seria interessante quando um botão for pressionado, o motor fique girando para um lado, e quando ele pressionar o outro, ele começa a girar para o outro lado.

Reply:
pelo que esta aqui, ele nao vai precisar ficar segurando, pq foi usado o when e nao o while

Reply:
vendo os comandos, percebi que os dois são comandos opostos para o intake, ou seja, se fosse possivel os dois rodarem ao mesmo tempo, ia quebrar o motor. mas para que isso nao aconteça, tem o required, que impede que outro comando use o subsystem enqaunto este esta sendo usado. mas existem formas melhores de fazer isso.

https://docs.wpilib.org/pt/stable/docs/software/commandbased/binding-commands-to-triggers.html